### PR TITLE
fixes #729. remove ref to deployer

### LINKED ISSUE #729: Bad link in HACKING.md
In 
https://github.com/openshift/origin-aggregated-logging/blob/master/HACKING.md, 

there is a link that points to 

https://github.com/openshift/origin-aggregated-logging/blob/master/deployer/deployer.yaml 

which doesn't exist (and return 404).

### PR DESCRIPTION
**Pull request by jcantrill**
fixes #729 by removing ref to the deployer

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ This will start the builds in the background. Use oc logs -f build/<component> t
 follow the build logs, or use oc start-build --follow <component> to run the build 
 in the foreground and follow the build logs.
 
-In order to run a deployment with these images, you would process the
-[deployer template](deployer/deployer.yaml) with the `IMAGE_PREFIX`
-parameter pointing to the cluster registry location (e.g `IMAGE_PREFIX=172.30.90.128:5000/logs/`).
+In order to run deploy logging with these images, you set the
```
Comment (richm) [inline HACKING.md]: remove `run`

File: HACKING.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ This will start the builds in the background. Use oc logs -f build/<component> t
 follow the build logs, or use oc start-build --follow <component> to run the build 
 in the foreground and follow the build logs.
 
-In order to run a deployment with these images, you would process the
-[deployer template](deployer/deployer.yaml) with the `IMAGE_PREFIX`
-parameter pointing to the cluster registry location (e.g `IMAGE_PREFIX=172.30.90.128:5000/logs/`).
+In order to run deploy logging with these images, you set the 
+[openshift-installer](https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging) 
+inventory varible `openshift_logging_install_logging` to the cluster registry location (e.g `172.30.90.128:5000/logs/`).
```
Comment (richm) [inline HACKING.md]: `variable`

Comment (richm) [reply HACKING.md]: I don't think this is right - `openshift_logging_install_logging` is a boolean?

Comment (ewolinetz) [reply HACKING.md]: correct, its a boolean

File: HACKING.md [hunk 3]
```diff
@@ -35,9 +35,9 @@ This will start the builds in the background. Use oc logs -f build/<component> t
 follow the build logs, or use oc start-build --follow <component> to run the build 
 in the foreground and follow the build logs.
 
-In order to run a deployment with these images, you would process the
-[deployer template](deployer/deployer.yaml) with the `IMAGE_PREFIX`
-parameter pointing to the cluster registry location (e.g `IMAGE_PREFIX=172.30.90.128:5000/logs/`).
+In order to deploy logging with these images, you set the 
+[openshift-installer](https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging) 
+inventory variable `openshift_logging_install_logging` to the cluster registry location (e.g `172.30.90.128:5000/logs/`).
```
Comment (richm) [inline HACKING.md]: Is this correct?  I thought `openshift_logging_install_logging` was a boolean?

Comment (ewolinetz) [reply HACKING.md]: correct, its a boolean

**Pull request merged**